srcpkgs/python3-PyHamcrest/template

```diff
@@ -11,7 +11,7 @@ maintainer="Orphaned <orphan@voidlinux.org>"
 license="BSD-3-Clause"
 homepage="https://github.com/hamcrest/PyHamcrest"
```

Comment:
Add a raw version of this to $changelog https://github.com/hamcrest/PyHamcrest/blob/main/CHANGELOG.rst